### PR TITLE
#161545625 add gitignore file

### PR DESCRIPTION
**Pull request by koechkevin**
#### What does this PR do?
- updates `.gitignore` file for this repository to a standard javascript gitignore file.
#### Description of Task to be completed?
- add a standard javascript gitignore file for this repository.
#### How should this be manually tested?
- N/A
#### Any background context you want to provide?
- The gitignore file can still be modified by contributors of this repository
#### What are the relevant pivotal tracker stories?
- [#161545625](https://www.pivotaltracker.com/story/show/161545625)

### CODE REVIEW

**Review by michael-basweti — CHANGES_REQUESTED** [review]
screen shot 2018-10-30 at 11 48 51
please check on the spelling of contributors, it is written as 'contriburors'

File: .gitignore [hunk 1]
```diff
@@ -10,18 +19,52 @@
 /build
 
 # misc
+.sass-cache
+connect.lock
 .DS_Store
+.env
```
Comment (codingedward) [inline .gitignore]: .env is actually tracked in Git.

**Review by codingedward — COMMENTED** [review]
Hello, please fix the highlighted issue.

File: .gitignore (hunk 1 above)
Comment (koechkevin) [reply .gitignore]: noted

**Review by michael-basweti — APPROVED** [review]

**Review by Arthur236 — APPROVED** [review]
Looks good 👍

**Pull request merged**